src/main/java/org/sonarqube/A.java

```diff
@@ -1,4 +1,14 @@
 package org.sonarqube;
 
 public class A {
+  String message = "foo";
+
+  // fix this
+  public String foo() {
+    message = null;
+    message.length();
```

Comment:
###AWABymUcAPaUDaFo_Lp4###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_1@9
The return value of "length" must be used.